src/app/sb-admin/components/pages/sb-user/sbuser.component.ts

```diff
@@ -48,33 +51,44 @@ export class SbUserComponent implements OnInit {
             { field: "phone", header: "Phone" },
             { field: "status", header: "Status" },
             { field: "channel", header: "Channel" },
-          ];
+        ];
 
         this.statuses = [
             { label: 'ACTIVE', value: 'active' },
             { label: 'INACTIVE', value: 'inactive' }
         ];
 
-        this.organizations = [
-            { name: 'NULP', code: 'NULP' },
-            { name: 'SHIKSHALOKAM', code: 'SHIKSHALOKAM' },
-            { name: 'TEKDI_TEST', code: 'TEKDI_TEST' }
-        ];
     }
 
-    openNew() {
-        this.user = {};
-        this.submitted = false;
-        this.userDialog = true;
+    getUserList() {
+        const body = {
+            "request": {
+                "filters": {
+                    "rootOrgId": ["0136268742469222406"]
```

Comment:
why this org id in the code?

Reply:
@snehalsabade remove this org id immediately

Reply:
its a data security risk

Reply:
Remove org id